app/components/Search/SearchResults.js

```diff
@@ -33,6 +34,13 @@ const SearchResultItem = ({
           <Icon className={styles.searchResultItemIcon} name={result.icon} />
         )}
       {result.label}
+      {result.date && (
+        <Time
+          time={Date.parse(result.date)}
```

Comment:
It is enough to just write `time={result.date}`, since it is converted to the proper moment-object (with "our" timezone) inside the `<Time />` component.